addons/l10n_fi/data/account_tax_report_line.xml

```diff
@@ -0,0 +1,115 @@
+        <field name="code">PURCHASE_SERVICE_EU_306</field>
+        <field name="tag_name">306</field>
+        <field name="sequence">5</field>
+        <field name="country_id" ref="base.fi"/>
+    </record>
+    <record id="account_financial_report_line_fi_import_payable_340" model="account.tax.report.line">
+        <field name="name">Vero tavaroiden maahantuonneista EU:n ulkopuolelta</field>
+        <field name="code">IMPORT_PAYABLE_340</field>
+        <field name="tag_name">340</field>
+        <field name="sequence">6</field>
+        <field name="country_id" ref="base.fi"/>
+    </record>
+    <record id="account_financial_report_line_fi_purchase_construct_318" model="account.tax.report.line">
+        <field name="name">Vero rakentamispalvelun ja metalliromun ostoista (käännetty verovelvollisuus)</field>
+        <field name="code">PURCHASE_CONSTRUCT_318</field>
+        <field name="tag_name">318</field>
+        <field name="sequence">7</field>
+        <field name="country_id" ref="base.fi"/>
+    </record>
+    <record id="account_financial_report_line_fi_purchase_307" model="account.tax.report.line">
+        <field name="name">Verokauden vähennettävä vero</field>
+        <field name="code">PURCHASE_307</field>
+        <field name="tag_name">307</field>
+        <field name="sequence">8</field>
+        <field name="country_id" ref="base.fi"/>
+    </record>
+    <record id="account_financial_report_line_fi_total_tax_payable" model="account.tax.report.line">
+        <field name="name">Maksettava vero / Palautukseen oikeuttava vero (-)</field>
+        <field name="sequence">9</field>
+        <field name="formula">SALE24_301 + SALE14_302 + SALE10_303 + PURCHASE_MATERIAL_EU_305 + PURCHASE_SERVICE_EU_306 + IMPORT_PAYABLE_340 + PURCHASE_CONSTRUCT_318 - PURCHASE_307</field>
+        <field name="country_id" ref="base.fi"/>
+    </record>
+    <record id="account_financial_report_line_fi_sale0_304" model="account.tax.report.line">
+        <field name="name">0-verokannan alainen liikevaihto</field>
+        <field name="tag_name">304</field>
+        <field name="sequence">11</field>
+        <field name="country_id" ref="base.fi"/>
+    </record>
+    <record id="account_financial_report_line_fi_sale_material_eu_311" model="account.tax.report.line">
+        <field name="name">Tavaroiden myynnit muihin EU-maihin</field>
+        <field name="tag_name">311</field>
+        <field name="sequence">12</field>
+        <field name="country_id" ref="base.fi"/>
+    </record>
+    <record id="account_financial_report_line_fi_sales_service_eu_312" model="account.tax.report.line">
+        <field name="name">Palvelujen myynnit muihin EU-maihin</field>
+        <field name="tag_name">312</field>
+        <field name="sequence">13</field>
+        <field name="country_id" ref="base.fi"/>
+    </record>
+    <record id="account_financial_report_line_fi_purchase_service_eu_base_306" model="account.tax.report.line">
```

Comment:
I think that the tax report line "Purchases of goods from other EU Member States" is missing, according to the document I found online: https://www.vero.fi/en/About-us/contact-us/forms/descriptions/selfassessed_tax_return_4001/ (See "Self-assessed tax return (pdf, 773 kB)").